### ISSUE TITLE
Hardcoded default marker set label name

### ISSUE THREAD

**Issue by ensirius** [webbukkit/DynmapCore#86]
![example](http://i.imgur.com/S9FKFbZ.png)

Currently is not possible to change default marker set label "Markers".

**Comment by Pisi-Deff** [comment]
True, it's not currently possible to change it in the main config.
You can, however, edit the existing `/plugins/dynmap/markers.yml` file and change the label there.

![image](https://user-images.githubusercontent.com/1923619/29209874-77ad9544-7e99-11e7-9f2c-cc9028024074.png)
